src/core/schemaConvertor.ts

```diff
@@ -248,4 +262,12 @@ export default class SchemaConvertor {
             this.processor.output(' */ ');
         }
     }
+
+    private addExport(id: SchemaId) {
+        const name = this.getLastTypeName(id);
+        const schemaRef = id.getJsonPointerHash();
+        const names = this.convertor(id);
+        const path = names.join('.');
+        this.exportedTypes.push({ name, path, schemaRef });
```

Comment:
I think the required data is only `SchemaId`.
If so, you can use the `ReferenceResolver$getAllRegisteredSchema`.